pages/parent/signup.tsx

```diff
@@ -0,0 +1,516 @@
+    const [artTherapy, setArtTherapy] = useState(false);
+    const [participantTherapy, setParticipantTherapy] = useLocalStorage(
+        "participantTherapy",
+        [],
+    );
+    const [hasOtherTherapy, setHasOtherTherapy] = useState(false);
+    const [otherTherapy, setOtherTherapy] = useLocalStorage("otherTherapy", "");
+
+    // Parent/guardian expectations
+    const [guardianExpectations, setGuardianExpectations] = useLocalStorage(
+        "guardianExpectations",
+        "",
+    );
+
+    // Emergency contact info
+    const [emergFirstName, setEmergFirstName] = useLocalStorage(
+        "emergFirstName",
+        "",
+    );
+    const [emergLastName, setEmergLastName] = useLocalStorage(
+        "emergLastName",
+        "",
+    );
+    const [emergNumber, setEmergNumber] = useLocalStorage("emergNumber", "");
+    const [emergRelationship, setEmergRelationship] = useLocalStorage(
+        "emergRelationship",
+        "",
+    );
+
+    // Parent info
+    const [parentFirstName, setParentFirstName] = useLocalStorage(
+        "parentFirstName",
+        "",
+    );
+    const [parentLastName, setParentLastName] = useLocalStorage(
+        "parentLastName",
+        "",
+    );
+    const [parentPhoneNumber, setParentPhoneNumber] = useLocalStorage(
+        "parentPhoneNumber",
+        "",
+    );
+    const [parentRelationship, setParentRelationship] = useLocalStorage(
+        "parentRelationship",
+        "",
+    );
+
+    // Heard from
+    const [heardFromFriendsAndFam, setHeardFromFriendsAndFam] = useState(false);
+    const [heardFromFlyers, setHeardFromFlyers] = useState(false);
+    const [heardFromEmail, setHeardFromEmail] = useState(false);
+    const [heardFromSocialMedia, setHeardFromSocialMedia] = useState(false);
+    const [heardFromOther, setHeardFromOther] = useState(false);
+    const [heardFromOptions, setHeardFromOptions] = useLocalStorage(
+        "heardFromOptions",
+        [],
+    );
+
+    const medicationDetails = isOnMedication ? (
+        <Box mt={4}>
+            <FormControl id="details">
+                <FormLabel>Please provide any details if necessary</FormLabel>
+                <Input
+                    placeholder="Details"
+                    onChange={(e) => setMedication(e.target.value)}
+                    value={medication}
+                />
+            </FormControl>
+        </Box>
+    ) : null;
+
+    const allergyDetails = hasAllergies ? (
+        <Box mt={4}>
+            <FormControl id="details">
+                <FormLabel>Please provide any details if necessary</FormLabel>
+                <Input
+                    placeholder="Details"
+                    onChange={(e) => setAllergies(e.target.value)}
+                    value={allergies}
+                />
+            </FormControl>
+        </Box>
+    ) : null;
+
+    const otherDifficultyDetails = hasOtherDifficulties ? (
+        <Box mt={4}>
+            <FormControl id="difficulty details">
```

Comment:
For id props in general, we separate words with a dash, otherwise, it semantically means it has two ids, which is not what we want. This applies to all the other occurrences below.

```suggestion
            <FormControl id="difficulty-details">
```